openlibrary/components/LibraryExplorer/components/LibraryToolbar.vue

```diff
@@ -499,7 +499,12 @@ export default {
 
       background: linear-gradient(to bottom, #fff, #ebdfc5 150%);
     }
-    .tab-bar > div {
+    .tab-bar > button {
```

Comment:
The main benefit for me is CSS readability; this is more clearly each "tab" component of the tab bar, whereas `.tab-bar > button` isn't as clear if you're just looking at the CSS.

```suggestion
    .tab-bar--tab {
```